src/components/ReviewCard.jsx

```diff
@@ -0,0 +1,9 @@
+export default function ReviewCard({ title, category, reviewImgUrl }) {
+  return (
+    <article>
+      <h3>Title: {title}</h3>
```

Comment:
I'd consider removing the word 'title' as it isn't needed. The design will make this look like a title already 👍